### DISCUSSION TITLE
How do i change to static ip?

### DISCUSSION THREAD

**Discussion by NetEyes** [telekom-security/tpotce#853] [Q&A]
Installed the latest tpot.iso.

How do i change to static ip?

**Comment by Tango9191** [comment]
Configure  /etc/network/interfaces

### Example static ip config
### Replace <eth0> with the name of your physical interface name
#
auto eth0
iface eth0 inet static
address 192.168.1.1
netmask 255.255.255.0
network 192.168.1.0
broadcast 192.168.1.255
gateway 192.168.1.1
dns-nameservers 192.168.1.1

Then reboot.

I had many problems after this, because e.g. mailoney or pf0 or heralding would not come up anymore.
If this happens, my solution was (example with heralding)

systemctl stop tpot
docker rmi $(docker images 'dtagdevsec/heralding' -a -q)
docker pull dtagdevsec/heralding:2006

**Reply by sudeyuksek** [reply]
When you are using static ip, is it bridged network or nat chosen as network?
![image](https://user-images.githubusercontent.com/60464361/178300609-8db1c6ae-764f-479f-b64a-7fcf2a2883a9.png) I changed etc/network/interfaces file as static. @Tango9191 @NetEyes

**Comment by goergiosdaniil** [comment]
Do we have anything here that is working? How can I set a static ip with specific steps?
